app/main.py

```diff
@@ -1 +1,50 @@
-#  write your code here
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        upd_appetite = self.appetite
+        if self.is_hungry is True:
+            self.is_hungry = not self.is_hungry
+            print(f"Eating {self.appetite} food points...")
+            self.appetite = 0
+            return upd_appetite
+        return 0
+
+
+class Cat(Animal):
+    def __init__(self, name: str, is_hungry: bool = True) -> None:
+        super().__init__(name, appetite=3, is_hungry=is_hungry)
+
+    @staticmethod
+    def catch_mouse() -> None:
+        print("The hunt began!")
+
+
+class Dog(Animal):
+    def __init__(self, name: str, is_hungry: bool = True) -> None:
+        super().__init__(name, appetite=7, is_hungry=is_hungry)
+
+    @staticmethod
+    def bring_slippers() -> None:
+        print("The slippers delivered!")
+
+
+def feed_animals(animals: list[Animal]) -> int:
+    sum_of_appetite = sum(
+        [animal.appetite for animal in animals if animal.is_hungry is True]
+    )
+    for animal in animals:
+        animal.feed()
+    return sum_of_appetite
```

Comment:
You should write a generator expression inside of the `sum` where you will call the `feed` method for each `Animal` instance in the `animals` list. Write this method in one line, and do not use the `[]` brackets inside of the `sum`.